publishing.md

```diff
@@ -53,12 +53,10 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Distributing changes
+## Changes
 
-If you modify or extend OpenFisca, and if you distribute those changes, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, country package, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
+If you modify or extend OpenFisca, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork of the core package, and of any country package, extension or reform, on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
 
-"Distributing changes" means sharing with others, in any way, the changes you made to the software. Providing software that integrates your changes in any format (source code, package, etc.) is considered as distribution, which means you also have to make your changes available.
+Please note that, under AGPL provisions, serving over the network is considered as publishing changes. You can read [this analysis](https://softwareengineering.stackexchange.com/questions/107883/agpl-what-you-can-do-and-what-you-cant/314908/) to get more insight on what you can and can't do with APGL licensed software.
```

Comment:
> Please note that, under AGPL provisions, serving over the network is considered as publishing changes.

If you _serve_ over the network, you're _propagating_ the software in a way it constitutes _conveying_.

The current language of the sentence is misleading, as serving over the network and publishing changes are two sources of propagation, but serving over the network does not constitute publication.

Why it could it be a problem? Someone could interpret it like this: _so if by serving over the network I'm already publishing changes, does it mean that I'm not required to make the source code of my changes available to the public by any other means than just serving the application?_.

Following the spirit of the changes proposed by this pull request, I propose we stick to the exegesis provided by the given link, i.e.:

> You can read [this analysis](https://softwareengineering.stackexchange.com/questions/107883/agpl-what-you-can-do-and-what-you-cant/314908/) to get more insight on what you can and can't do with APGL licensed software.

And the analysis is:

> You have to make available the modified source code of X (but not any other part of your overall application) if you distribute your own application and X has GPLv3 or AGPLv3. The same holds if you make available the service provided by your application via a network e.g. as a web application and X has AGPL.

I think the explanation given by the link is already pretty straight-forward, and potentially conflicting with this one, so I propose let's favour the former over the latter.